packages/aws-lambda/lambdas/bin/create-zip.sh

```diff
@@ -37,37 +37,41 @@ echo
 echo Building all local tar.gz files.
 
 if [[ -z "${BUILD_LAMBDAS_WITH-}" ]]; then
-  echo "Environmant variable BUILD_LAMBDAS_WITH has not been provided, assuming \"npm\" (build with latest npm package)."
+  echo "Environment variable BUILD_LAMBDAS_WITH has not been provided, assuming \"local\"."
+  BUILD_LAMBDAS_WITH=local
 fi
 
 # We will want to install/uninstall local npm package tar files, which means we need to build them first. Note: Even for
 # options BUILD_LAMBDAS_WITH=layer or BUILD_LAMBDAS_WITH=npm, we need to build the packages, because we for all three
 # packages we will call "npm uninstall -S $package-name"  and if the package.json points to the tar file it needs
 # to exist so npm can uninstall it and all its transitive dependencies.
+
+version=$(jq -r '.version' package.json)
+
 echo "Building local tar.gz for @instana/serverless."
 cd ../serverless
 npm --loglevel=warn pack
-mv instana-serverless-*.tgz ../aws-lambda/instana-serverless.tgz
+mv instana-serverless-${version}.tgz ../aws-lambda/instana-serverless.tgz
```

Comment:
Its better to read the exact version. Because sometimes you have multiple tgz files and then the "mv" cmd does not work. Have seen this a couple of times.